spring-context/build.gradle

```diff
@@ -19,6 +18,8 @@ dependencies {
     testImplementation mn.micronaut.inject.java
     testImplementation projects.springAnnotation
     testImplementation mn.micronaut.inject.java.test
+    testImplementation "org.springframework:spring-jdbc:5.3.20"
```

Comment:
Do we need an explicit version? If yes that it would be better to have it in the versions toml.

Reply:
I tried this but hit a bug with version catalogs that @melix and I couldn't resolve